File: user-backend/Common/Models/ManagementApp/Reports/FreeTrialUserReport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Common.Models.ManagementApp.Reports
{
    public class FreeTrialUserReport
    {
        public string Name { get; set; }
        public string Email { get; set; }
        public string UserName { get; set; }
        public int DocumentsUsage { get; set; }
        public int SMSUsage { get; set; }
        public int TemplatesUsage { get; set; }
        public DateTime CreationDate { get; set; }
        public DateTime ExpirationDate { get; set; }

    }
}

File: wesignmanagement-client-DEV/src/app/models/results/Reports/freeTrialUserReport.ts
export class freeTrialUserReport {
    public Name: string;
    public email: string;
    public userName: string;
    public documentsUsage: number = 0;
    public smsUsage: number = 0;
    public templatesUsage: number = 0;
    public creationDate: Date = new Date();
    public expirationDate: Date = new Date();
     
}

export class freeTrialUserReports {
    public freeTrialUsersReports: freeTrialUserReport[];
}